docker/dev.docker-compose.yml

```diff
@@ -0,0 +1,141 @@
+version: "2"
+services:
+  supervisord:
+    build:
+      context: ..
+      dockerfile: docker/dev.Dockerfile
+    volumes:
+      - ..:/assembl
+    command: "sh -c '. venv/bin/activate && supervisord -n'"
+    depends_on:
+      - database
+      - redis
+      - memcached
+      - elasticsearch
+    ports:
+      - "6543:6543"
+      - "8080:8080"
+  dev:
+    image: assembl_supervisor
+    volumes:
+      - ..:/assembl
+    command: "sh -c '. venv/bin/activate && supervisorctl start dev:'"
+    depends_on:
+      - supervisord
+    restart: always
```

Comment:
You probably should add
```
supervisor__autostart_webpack = true
supervisor__autostart_pserve = true
supervisor__autostart_gulp = true
```
in your `docker_develop.rc` instead of starting a container to `start dev:`

Reply:
will do